README.md

```diff
@@ -17,7 +17,7 @@ To locally preview appium.io
 1. Node + NPM
 1. Ruby Bundler (`gem install bundler`)
 1. pip (`brew install pip`)
-1. MkDocs 0.16.3 (`pip install mkdocs==0.16.3`)
+1. MkDocs (`pip install mkdocs`)
```

Comment:
why it was necessary to limit the version?

Reply:
It wasn't working before so I pinned it to 0.16.3

The fix was to get rid of the `meta` variable and now it works fine with 0.17